gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/io/gravitee/rest/api/service/impl/TagService_FindIdsByAndReferenceTest.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.rest.api.service.impl;
+
+import io.gravitee.repository.exceptions.TechnicalException;
+import io.gravitee.repository.management.api.TagRepository;
+import io.gravitee.repository.management.model.Tag;
+import io.gravitee.repository.management.model.TagReferenceType;
+import io.gravitee.rest.api.service.AuditService;
+import io.gravitee.rest.api.service.GroupService;
+import io.gravitee.rest.api.service.exceptions.TagNotFoundException;
+import io.gravitee.rest.api.service.v4.ApiTagService;
+import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class TagService_FindIdsByAndReferenceTest {
+
+    private static final String ORG_ID = "DEFAULT";
+
+    @InjectMocks
+    private TagServiceImpl tagService = new TagServiceImpl();
+
+    @Mock
+    private TagRepository tagRepository;
+
+    @Mock
+    private ApiTagService apiTagService;
+
+    @Mock
+    private AuditService auditService;
+
+    @Mock
+    private GroupService groupService;
+
+    @BeforeEach
+    public void init() {
+        reset(tagRepository);
+    }
+
+    @Test
+    public void should_not_throw_exception_if_empty_tag_ids_parameter() throws TechnicalException {
+        when(tagRepository.findByIdsAndReference(new HashSet<>(), ORG_ID, TagReferenceType.ORGANIZATION)).thenReturn(Set.of());
+        try {
+            tagService.checkTagsExist(new HashSet<>(), ORG_ID, io.gravitee.rest.api.model.TagReferenceType.ORGANIZATION);
+        } catch (Exception e) {
+            fail("Should not throw exception");
+        }
```

Comment:
The try/catch is unnecessary, if an exception is thrown JUnit will catch it and flag the test as failed